### ISSUE TITLE
Landing pages return 404 error page not found

### ISSUE THREAD

**Issue by minhntvnc** [gophish/gophish#995]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
Lastest version: gophish-v0.5.0-linux-64bit.zip
Brief description of the issue: 
After import site, inaccessible landing pages with error 404
What are you expecting to see happen? :
See clone webpage
What are you seeing happen?
404 Error
If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:
Ex: Import from https://mail.vncert.vn
Insert code here
```

Please provide any terminal output that may be relevant below:
101.96.121.196 - - [08/Mar/2018:07:40:30 -0500] "GET /favicon.ico HTTP/1.1" 404 19 "http://mail1.XXX.com.vn/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0.3 Safari/604.5.6"
101.96.121.196 - - [08/Mar/2018:07:40:30 -0500] "GET / HTTP/1.1" 404 19 "" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0.3 Safari/604.5.6"
```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by mcvic1rj** [comment]
Have you sent a test message to yourself through GoPhish? The landing pages require a unique code to show up and that code is only active when the campaign is active

So having gophish.me.com, importing the site mail.me.com, then going to hxxps://gophish.me.com/ will not return the landing page

**Comment by jordan-wright** [comment]
We'll need quite a bit more information before we can help troubleshoot. So you're importing the page, have you launched a campaign? What did you put as the URL in the campaign settings? Did you make sure the links in the email template used the `{{.URL}}` template tag correctly?

And @mcvic1rj is absolutely correct- right now sending a test email doesn't give clickable links. You'll need to launch a campaign with just your email address to properly test the landing page. Eventually we'll find a way around this but it's a known limitation right now.

**Comment by koolkidjam23** [comment]
Hello Jordan I just want to say thank you for your hard work and what you do for the community. I am a huge fan of the software. I come on this page everyday looking for help lol but I always seem to find it on one of your old threads. 

Im having a MAJOR headache right now. iv been stuck on this situation for 2 weeks. 

I have Nginx configured on my VPS with gophish. Everything works perfect until I encrypt with certbot. It changes my configuration. and blocks landing pages. Saying theres a error in a tls Handshake. 

If you can please help me or just take a look at my config and see if I'm using the right ports I would appreciate so much.

**Comment by minhntvnc** [comment]
Landing page is ok now. I fix url so cannot launcher a landing page. Must {{.URL}} and define this variable at New Campaign. Thank you so much, bro!
